cypress/integration/task/test_table.ts

```diff
@@ -78,14 +81,11 @@ describe("Tests Table", () => {
       expect(loc.search).to.include(DESCEND_PARAM);
     });
   });
+
   describe("Test Status Selector", () => {
     beforeEach(() => {
       cy.visit(TESTS_ROUTE);
-      cy.get("[data-cy=test-status-select] > .cy-treeselect-bar").click();
-    });
-
-    it("Status select says 'No filters selected' by default", () => {
```

Comment:
This test is removed since the input dropdown is now an icon embedded in the table column